tox.ini

```diff
@@ -11,7 +11,7 @@ envlist =
     py{38,39,310,py38}-django40-mongo-alchemy-{sqlite,postgres}
     py310-djangomain-mongo-alchemy-{sqlite,postgres}
 
-toxworkdir = {env:TOX_WORKDIR:.tox}
```

Comment:
This has been wrong for a really long time, according to the [docs](https://tox.wiki/en/latest/config.html#injected-environment-variables) its `TOX_WORK_DIR`, not `TOX_WORKDIR`

Reply:
Nice catch, see https://github.com/FactoryBoy/factory_boy/pull/945.

Reply:
I'll undo this, and let you handle it on #945